internal/plumbing/renames.go

```diff
@@ -384,9 +389,8 @@ func (ra *RenameAnalysis) blobsAreClose(blob1 *CachedBlob, blob2 *CachedBlob) (b
 	cleanReturn := false
 	defer func() {
 		if !cleanReturn {
-			log.Println()
-			log.Println(blob1.Hash.String())
-			log.Println(blob2.Hash.String())
+			ra.l.Warnf("unclean return detected for blobs '%s' and '%s'\n",
```

Comment:
```suggestion
			ra.l.Warnf("\nunclean return detected for blobs '%s' and '%s'\n",
```
because of the progress bar